synapse/rest/client/v1/login.py

```diff
@@ -414,7 +415,7 @@ def handle_cas_response(self, request, cas_response_body, client_redirect_url):
                     raise LoginError(401, "Unauthorized", errcode=Codes.UNAUTHORIZED)
 
         return self._sso_auth_handler.on_successful_auth(
-            user, request, client_redirect_url
+            user, request, client_redirect_url, displayname,
```

Comment:
```suggestion
            user, request, client_redirect_url, displayname
```

This is from the liniting. You can use [`black`](https://github.com/psf/black) to autoformat btw